basics/gradle.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: content
+title: Gradle Integration
+author: wolfram
+as_version: 3.0
+cl_version: 1.0.2 or 1.1.0-beta5
+order: 6
+---
+
+### Top level build configuration
+You can add ConstraintLayout as you do with other dependencies: By adding it to your gradle `build.xml` file.
+
+Since ConstaintLayout is part of the Support Library family provided by Google, you have to use the [Google Maven repository](https://developer.android.com/studio/build/dependencies.html#google-maven).
+
+Most likely your project is using other parts of the Support Library family anyway, but to be sure, here's how to configure your top level build file to use the Google repository:
+
+```gradle
+buildscript {
+  repositories {
+    google()
+    jcenter()
+  }
+
+  dependencies {
+    classpath 'com.android.tools.build:gradle:3.0.1'
+  }
+}
+
+allprojects {
+  repositories {
+    google()
+    jcenter()
+  }
+}
+```
+
```

Comment:
Nit: extra whiteline

Reply:
I did use all those extra line breaks on purpose (apart from those at the end of the file). I consider the document to be easier readable for the editor with two line breaks where a new section starts. I always do that - unless it causes additional <p> or <br> tags to appear in the final document. But, if we use a different style here on constraintlayout.github.io, I'm fine with that and will remove them.